core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -197,8 +197,7 @@ class RestAPIVersion(config: WhiskConfig, apiPath: String, apiVersion: String)(
     private val triggers = new TriggersApi(apiPath, apiVersion)
     private val activations = new ActivationsApi(apiPath, apiVersion)
     private val rules = new RulesApi(apiPath, apiVersion)
-    private val webexp = new WebActionsApi(Seq("experimental", "web"), WebApiDirectives.exp)
-    private val web = new WebActionsApi(Seq("web"), WebApiDirectives.web)
+    private val web = new WebActionsApi(Seq("web"), WebApiDirectives)
 
```

Comment:
@rabbah I am having trouble accessing WebApiDirectives here now that I made your suggested edit to the constructor, any suggestions?